index.html

```diff
@@ -63,6 +64,8 @@ <h1>Dashboard</h1>
                 <h3>Saved Dashboards</h3>
                 <div id="dashboards-list"></div>
                 <button onclick="openDashboardCreateDialog()" class="btn-primary btn-icon-before"><i class="fa fa-plus"></i>ADD Dashboard</button>
+
+                <div id="github-dashboard-link">Code on: <a href="https://github.com/DirkHeinke/mqttDashboard">GitHub</a> </div>
```

Comment:
It might look a little bit better, when we add a little bit of padding (8px) and maybe the github icon?

Reply:
Added some space, but no icon.